.luacheckrc

```diff
@@ -0,0 +1,73 @@
+std             = "ngx_lua"
+unused_args     = false
+redefined       = false
+max_line_length = false
+
+
+globals = {
+    "_KONG",
+    "kong",
+    "ngx.IS_CLI",
+}
+
+
+not_globals = {
+    "string.len",
+    "table.getn",
+}
+
+
+ignore = {
+    "6.", -- ignore whitespace warnings
+}
+
+
+exclude_files = {
+    "spec/fixtures/invalid-module.lua",
+    "spec-old-api/fixtures/invalid-module.lua",
+}
+
+files["kong/tools/kong-lua-sandbox.lua"] = {
+     read_globals = {
+        "_ENV",
+        "table.pack",
+        "table.unpack",
+     }
+}
+
+
+files["kong/hooks.lua"] = {
+    read_globals = {
+        "table.pack",
+        "table.unpack",
+    }
+}
+
+
+files["kong/db/schema/entities/workspaces.lua"] = {
+    read_globals = {
+        "table.unpack",
+    }
+}
+
+
+files["kong/plugins/ldap-auth/*.lua"] = {
+    read_globals = {
+        "bit.mod",
+        "string.pack",
+        "string.unpack",
+    },
+}
```

Comment:
Nit: same as the other comment re: kong-specific config, though definitely less important since these won't match any in-repo files.

```suggestion
```